robottelo.properties.sample

```diff
@@ -303,6 +303,31 @@ ssh_key=
 # 'semanage port -a -t websm_port_t -p tcp <port-range>'
 port_range=9091, 14999
 
+# [ec2]
+# AWS EC2 to be added as a compute resource.
+# Access key ID of the IAM user
+# access_key=
```

Comment:
@san7ket , good point, these values can come in from CI, but now the point is these values should be masked, now we need to be sure , whether these masked values can be passed in here. I feel may be not. Need to figure a way to get past this.